app/serviceaccount.go

```diff
@@ -0,0 +1,392 @@
+			}
+		}
+		spec.Access.NamespaceAccesses = nrs
+	}
+
+	req := &authservice.UpdateServiceAccountRequest{
+		ServiceAccountId: serviceAccountID,
+		Spec:             spec,
+	}
+	if ctx.IsSet(ResourceVersionFlagName) {
+		req.ResourceVersion = ctx.String(ResourceVersionFlagName)
+	}
+	if ctx.IsSet(RequestIDFlagName) {
+		req.AsyncOperationId = ctx.String(RequestIDFlagName)
+	}
+	if req.ResourceVersion == "" {
+		req.ResourceVersion = sa.ResourceVersion
+	}
+
+	resp, err := c.client.UpdateServiceAccount(c.ctx, req)
+	if err != nil {
+		return fmt.Errorf("unable to update service account: %w", err)
+	}
+	return PrintProto(resp.AsyncOperation)
+}
+
+func NewServiceAccountCommand(getServiceAccountClientFn GetServiceAccountClientFn) (CommandOut, error) {
+	var c *ServiceAccountClient
+	return CommandOut{
+		Command: &cli.Command{
+			Name:    "service-account",
+			Aliases: []string{"sa"},
+			Usage:   "Service Account management operations",
+			Before: func(ctx *cli.Context) error {
+				var err error
+				c, err = getServiceAccountClientFn(ctx)
+				return err
+			},
+			Subcommands: []*cli.Command{
+				{
+					Name:    "list",
+					Usage:   "List service accounts",
+					Aliases: []string{"l"},
+					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:    pageTokenFlagName,
+							Usage:   "Page token for paging list service accounts request",
+							Aliases: []string{"p"},
+						},
+						&cli.IntFlag{
+							Name:    pageSizeFlagName,
+							Usage:   "Page size for paging list service accounts request",
+							Value:   10,
+							Aliases: []string{"s"},
+						},
+					},
+					Action: func(ctx *cli.Context) error {
+						return c.listServiceAccounts(ctx.String(pageTokenFlagName), ctx.Int(pageSizeFlagName))
+					},
+				},
+				{
+					Name:    "get",
+					Usage:   "Get service account information",
+					Aliases: []string{"g"},
+					Flags: []cli.Flag{
+						serviceAccountIDFlag,
+					},
+					Action: func(ctx *cli.Context) error {
+						sa, err := c.getServiceAccount(ctx.String(serviceAccountIDFlagName))
+						if err != nil {
+							return err
+						}
+						return PrintProto(sa)
+					},
+				},
+				{
+					Name:    "update",
+					Usage:   "Update service account from Temporal Cloud",
+					Aliases: []string{"u"},
+					Flags: []cli.Flag{
+						serviceAccountIDFlag,
+						serviceAccountDescriptionFlag,
+						serviceAccountNameFlag,
+						ResourceVersionFlag,
+						RequestIDFlag,
+					},
+					Action: func(ctx *cli.Context) error {
+						if len(ctx.String(serviceAccountDescriptionFlagName)) == 0 &&
+							len(ctx.String(serviceAccountNameFlagName)) == 0 {
+							return fmt.Errorf("no update parameters provided")
+						}
+						return c.performUpdate(
+							ctx,
+							ctx.String(serviceAccountIDFlagName),
+							ctx.String(serviceAccountNameFlagName),
+							ctx.String(serviceAccountDescriptionFlagName),
+							nil,
+							"",
+							nil,
+						)
+					},
+				},
+				{
+					Name:  "disable",
```

Comment:
We de-scoped the disable functionality for the short-term, and the new proto files have an update around this. Should be able to remove it now.

Reply:
done